### PR TITLE
Removed text block.

### PR DESCRIPTION
**Pull request by mickel1138**
This may be a bit aggressive, but it works.

### CODE REVIEW

File: assets/js/vue/components/search/SearchForm.vue [hunk 1]
```diff
@@ -76,20 +76,7 @@
                 @noOfResults="changeNoOfResults"
                 :formValues="form"/>
         <section class="section-content pt-3 bg" v-else>
-            <div class="container">
-                <article class="card">
-                    <div class="card-body text-center">
-                        <h5 class="card-title">
-                            GRIIDC Datasets
```
Comment (praneethpr) [inline assets/js/vue/components/search/SearchForm.vue:83]: Removing this would remove the text on GRIIDC search as well, we need a flag variable to switch

Comment (mickel1138) [reply assets/js/vue/components/search/SearchForm.vue]: Made it switchable.

File: templates/nas-grp-base.html.twig [hunk 2]
```diff
@@ -159,6 +159,10 @@
         });
     </script>
 
+    <script>
+        window.PELAGOS_TEMPLATE_PROPS = {"showSearchTextBlock": false};
```
Comment (praneethpr) [inline templates/nas-grp-base.html.twig:163]: Needs to be set to TRUE in the griidc template

Comment (fightingtexasaggie) [reply templates/nas-grp-base.html.twig:163]: yes, currently the search page doesn't show at all under the griidc template.

Comment (mickel1138) [reply templates/nas-grp-base.html.twig:163]: I'm just gonna flip it around.

File: assets/js/vue/components/search/SearchForm.vue [hunk 3]
```diff
@@ -76,7 +76,7 @@
                 @noOfResults="changeNoOfResults"
                 :formValues="form"/>
         <section class="section-content pt-3 bg" v-else>
-            <div class="container">
+            <div v-show="!hideSearchTextBlock" class="container">
```
Comment (praneethpr) [inline assets/js/vue/components/search/SearchForm.vue:79]: using the same name as the window variable key is kind of confusing. Should we name the function something else?

Comment (praneethpr) [reply assets/js/vue/components/search/SearchForm.vue:79]:
```suggestion
            <div v-show="!displayTextBlock" class="container">
```

File: assets/js/vue/components/search/SearchForm.vue [hunk 4]
```diff
@@ -117,6 +117,15 @@
                 submitted: false,
             }
         },
+        computed: {
+            hideSearchTextBlock: function () {
```
Comment (praneethpr) [inline assets/js/vue/components/search/SearchForm.vue:121]:
```suggestion
            displayTextBlock: function () {
```

**Pull request merged**